chart/values.yaml

```diff
@@ -110,6 +110,10 @@ kube-prometheus-stack:
     additionalRuleLabels:
       alertgroup: Cluster
 
+    additionalRuleGroupLabels:
```

Comment:
Reference a related issue here, so we don't forget what it's about.

Reply:
A reference to a related issue has been added.